### PR TITLE
Fix for Task 1078

### PR DESCRIPTION
**Pull request by sujata-m**
- Fix for Task [1078](https://dev.azure.com/TDEI-UW/TDEI/_workitems/edit/1078)
- Added the functionality to stop processing further if there are 20 errors(default)
- max_errors can be passed as a parameter to validate function
- Added unit test cases
- Updated README.md
- Updated CHANGELOG.md
- Updated Package version
- Added Github action for running test cases

### CODE REVIEW

**Comment by susrisha** [comment]
Approving after checking the unit tests. All unit tests are successful.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Change log
 
+### 0.2.2
+- Added functionality to get the specific umber of errors
```
Comment (susrisha) [inline CHANGELOG.md:4]: its `number` . Please check typo
- Added functionality to return after a specific number of errors

File: tests/unit_tests/test_osw_validation.py [hunk 2]
```diff
@@ -76,6 +76,20 @@ def test_invalid_zipfile_with_schema(self):
         self.assertFalse(result.is_valid)
         self.assertIsNotNone(result.errors)
 
+    def test_invalid_zipfile_default_error_count(self):
+        validation = OSWValidation(zipfile_path=self.invalid_zipfile)
+        result = validation.validate()
+        self.assertFalse(result.is_valid)
+        self.assertIsNotNone(result.errors)
+        self.assertEqual(20, len(result.errors))
```
Comment (susrisha) [inline tests/unit_tests/test_osw_validation.py:84]: probably check with lessthan or equal

**Review by susrisha — CHANGES_REQUESTED** [review]

File: tests/unit_tests/test_osw_validation.py (hunk 2 above)
Comment (sujata-m) [reply tests/unit_tests/test_osw_validation.py:84]: okay

File: CHANGELOG.md (hunk 1 above)
Comment (sujata-m) [reply CHANGELOG.md:4]: my bad

**Review by susrisha — APPROVED** [review]

**Pull request merged**